content/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -1,42 +1,52 @@
 ---
 title: ChainSync
 weight: 3
-description: ChainSync - synchronizing the Blockchain
 dashboardWeight: 2
 dashboardState: reliable
 dashboardAudit: missing
 dashboardTests: 0
 ---
 
-# ChainSync - synchronizing the Blockchain
-
-## What is blockchain synchronization?
+# ChainSync
 
 Blockchain synchronization ("sync") is a key part of a blockchain system.
 It handles retrieval and propagation of blocks and transactions (messages), and
 thus is in charge of distributed state replication.
-**This process is security critical -- problems here can be catastrophic to the
-operation of a blockchain.**
+As such, this process is security critical -- problems with state replication can have severe consequences to the
+operation of a blockchain.
 
-## What is ChainSync?
+## ChainSync Overview
 
 `ChainSync` is the protocol Filecoin uses to synchronize its blockchain. It is
 specific to Filecoin's choices in state representation and consensus rules,
 but is general enough that it can serve other blockchains. `ChainSync` is a
 group of smaller protocols, which handle different parts of the sync process.
 
+Chain synchronisation generally applies to three main processes:
+1. Chain synchronization when a node first joins the network and needs to get to the current state before validating or extending the chain.
+2. Chain synchronisation when a node has fell out of sync, e.g., due to a brief disconnection.
+3. Continuous chain synchronisation to keep up with the latest messages and blocks.
+
+There are three main protocols used to achieve synchronisation for these three processes.
+- `GossipSub` is the libp2p pubsub protocol used to propagate messages and blocks. It is mainly used in the third process above when a node needs to stay in sync with new blocks being produced and propagated.
+- `BlockSync` is used to synchronise specific parts of the chain, that is from and to a specific height.
+- `Bitswap` is used to request and receive blocks, when a node is synchonized but GossipSub has failed to deliver some blocks to a node.
+- `GraphSync` can be used to fetch parts of the blockchain as a more efficient version of `Bitswap`.
```

Comment:
These I think are secondary in the sync process, we may be able to remove them or list them separately to keep the main list concise and easier to grasp. They don't fetch block headers (they might be used to fetch messages or complete chain information in some other way).